pkg/descriptions.json

```diff
@@ -69,6 +69,11 @@
     "CommonTypes": []
   },
   {
+    "Name": "Microsoft.NETFramework.Compatibility.Shims",
+    "Description": "Includes .NET Framework shims in order for Microsoft.NETFramework.Compatibility to find all the necessary types when running in .NET Core",
```

Comment:
Is this quite right? M.NF.C is not what's finding the types, I think.
`Includes .NET Framework shims that type forward to .NET Core implementations in the Microsoft.NETFramework.Compatibility package.` or something?

Reply:
That is better. :)